### ISSUE TITLE
Fixed functional tests: Several tests never run - class not found for tests in system.test.

### ISSUE THREAD

**Issue by indigoxela** [backdrop/backdrop-issues#5490]
## Description of the bug

While fixing something unrelated, it turned out that several tests actually never run:

`core/modules/system/tests/system.tests.info` contains wrong file paths

It's still not clear, why these tests ran in the past, but don't for several weeks. Eventually related: Issue #5415: Gracefully handle empty or malformed .tests.info files.

Note that on 7.4 and 8.1 the tests fail *silently*, but the error still occurs. On php 5.6 the error is not so silent anymore, probably because of a different setup, that's why the problem has been discovered. But the test *did* run December last year.

## Steps To Reproduce

1. Enable the Simpletest module
2. Enable error display
3. Try to run a test from core/modules/system/tests/system.test, for example BackdropSetMessageTest

## Actual behavior

```
    Warning: include_once(...core/modules/system/tests/tests/system.test): failed to open stream: No such file or directory in include_once() (line 172 of ...core/modules/simpletest/simpletest.module).
    Warning: include_once(): Failed opening '...core/modules/system/tests/tests/system.test' for inclusion (include_path='.:/usr/share/php') in include_once() (line 172 of ...core/modules/simpletest/simpletest.module).
    Error: Class 'BackdropSetMessageTest' not found in simpletest_run_tests() (line 173 of ...core/modules/simpletest/simpletest.module).
```

## Expected behavior

Successful test runs.

## Additional information

- Backdrop CMS version: latest dev
- Web server and its version: Apache with php-fpm

**Comment by indigoxela** [comment]
The problem has been introduced with https://github.com/backdrop/backdrop/pull/537, the only remaining file with `file = tests/XXX.test` is actually the system module (system.tests.info). The "tests/" prefix has been removed from all other files in the meantime.

It's still not clear, why this wrong path wasn't a problem until recently, maybe @bugfolder has an idea?

But it's easy to fix that. A [PR](https://github.com/backdrop/backdrop/pull/3942) is available for testing and review.
To test in the sandbox, follow the steps in the issue description. Any test from core/modules/system/tests/system.tests.info should run.

To compare the failures, get a *recent* dev version without that patch - tests from system.test die with a "Class not found" error.

**Comment by klonos** [comment]
Thanks for catching that @indigoxela ...and for figuring out the problem, and for fixing 🙏🏼 

Without the PR:
![image](https://user-images.githubusercontent.com/2423362/152325661-78561bde-53e1-4bff-81e7-80e2564258a4.png)

In the PR sandbox:
![image](https://user-images.githubusercontent.com/2423362/152325388-bbb7b865-59f4-4695-86a1-10be1504e513.png)

**Comment by bugfolder** [comment]
> It's still not clear, why this wrong path wasn't a problem until recently

The timing and subject suggests it's related to PR https://github.com/backdrop/backdrop/pull/3876, though I can't see any obvious reason in the code why this would happen. But it would probably be a good thing to add an explicit check for existence of the "file = ..." entry in the .info file.

**Comment by quicksketch** [comment]
Thanks @indigoxela, @klonos, and @bugfolder! I merged https://github.com/backdrop/backdrop/pull/3942 to get this problem fixed so we can properly test PHP 5.6 in https://github.com/backdrop/backdrop-issues/issues/3992.
